subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/InstrumentationTestsAction.kt

```diff
@@ -61,9 +61,7 @@ internal class InstrumentationTestsAction(
 
         val testsExecutionResults = scheduler.schedule()
 
-        finalizer.finalize(
-            testsExecutionResults = testsExecutionResults
-        )
+        finalizer.finalize(testsExecutionResults = testsExecutionResults)
```

Comment:
I'd like to choose unambiguous and simple rules for suc formatting.
I see many changes in both ways in PRs. It makes harder to review.